cowprotocol/accounting/slippage/slippage_per_transaction_4070059.sql

```diff
@@ -0,0 +1,33 @@
+-- This query return slippage per transaction
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
+-- - block_time: time of settlement transaction
+-- - tx_hash: settlement transaction hash
+-- - solver_address: address of the solver executing the settlement
+-- - slippage_usd: USD value of slippage
+-- - slippage_native: value of slippage in native token
+--
+-- Results of the query are filtered to not include batches from excluded_batches.
+-- Batches are also excluded if there is a non-zero imbalance and no value (in native atomes).
+
+with excluded_batches as (
+    select tx_hash from query_3490353
+)
+
+select
+    s.block_time,
+    s.tx_hash,
+    solver_address,
```

Comment:
i guess solver_address could be skipped here as it is not really relevant to the main functionality of the query.

Reply:
It was useful for me to have a query with results per transaction and where I could filter on solver address. A cleaner way might be what I proposed in the response above.